tools/mtouch/xlf/Errors.pl.xlf

```diff
@@ -2986,6 +2986,14 @@
         <note>
 		</note>
       </trans-unit>
+      <trans-unit id="MX2111">
+        <source>Can not find the corlib assembly '{0}' in the list of loaded assemblies.
+		</source>
+        <target state="new">Can not find the corlib assembly '{0}' in the list of loaded assemblies.
+		</target>
+        <note>
+		</note>
```

Comment:
Nit: what is going on with the indentation?

Reply:
No idea, this file is generated, and there are other entries with funny indentation as well.